symbolicationWebService.py

```diff
@@ -125,42 +159,24 @@ def do_POST(self):
         return
 
       LogDebug("Request body: " + requestBody)
-      rawRequest = json.loads(requestBody)
 
-      request = SymbolicationRequest(gSymFileManager, rawRequest)
-      if not request.isValidRequest:
+      response = yield gPool.submit(processSymbolicationRequest, requestBody)
+      if response is None:
         LogDebug("Unable to parse request")
```

Comment:
For only the `RequestHandler` log calls, not so much, but across the whole app (including the child process), quite complex, because the async nature of the code.